cmd/podman/images.go

```diff
@@ -87,7 +87,7 @@ var (
 	imagesCommand     cliconfig.ImagesValues
 	imagesDescription = "lists locally stored images."
 
-	_imagesCommand = &cobra.Command{
+	_imagesCommand = cobra.Command{
```

Comment:
This surprised me: everywhere else, the convention is
```
    _xxxCommand = &cobra.Command{
```
It's even more jarring because of the corresponding change in `main.go`.

Reply:
it is the black magic to get the subcommand image to honor images, ls, list but not honor ls, list on the main podman menu.  Perhaps better put:

* podman image ls should work
* podman ls should not work